Source/ThirdParty/SDL/src/sensor/android/SDL_androidsensor.c

```diff
@@ -171,7 +171,7 @@ SDL_ANDROID_SensorUpdate(SDL_Sensor *sensor)
     ASensorEvent event;
     struct android_poll_source* source;
 
-    if (ALooper_pollAll(0, NULL, &events, (void**)&source) == LOOPER_ID_USER) {
+    if (ALooper_pollOnce(0, NULL, &events, (void**)&source) == LOOPER_ID_USER) {
```

Comment:
Is this change related to Diligent?

Reply:
It's related to supporting newer androids, gonna keep it just because why not it's mainstream SDL commit